source/platforms/git/localGetCommits.ts

```diff
@@ -14,30 +14,33 @@ const authorDate = "%ai"
 const committerName = "%cn"
 const committerEmail = "%ce"
 const committerDate = "%ci"
-const message = "%s" // this is subject, not message, so it'll only be one line
+const message = "%f" // this is subject, not message, so it'll only be one line
```

Comment:
@orta this is the crux of it - curious if you're comfortable with this solution?

Relevant doc for reference: https://git-scm.com/docs/pretty-formats

Reply:
Yep, seems totally fine to me!